client/components/command-pallette/wpcom-command-pallette.tsx

```diff
@@ -213,6 +227,17 @@ export const WpcomCommandPalette = () => {
 		>
 			<StyledCommandsMenuContainer className="commands-command-menu__container">
 				<Command label={ __( 'Command palette' ) } onKeyDown={ onKeyDown }>
+					<div>
+						{ selectedCommandName && (
+							<PaletteBreadCrumb>
+								<StyledCommandsTitle>Commands / </StyledCommandsTitle>
+								<div>
+									{ '\u00A0' }
```

Comment:
Let's use `&nbsp;` instead of the unicode version.

```suggestion
									&nbsp;
```

Reply:
Thanks, I made the update